src/views/FarmAuction/components/AuctionLeaderboard/AuctionRibbon.tsx

```diff
@@ -0,0 +1,55 @@
+import { Text } from '@pancakeswap/uikit'
+import { Auction, AuctionStatus } from 'config/constants/types'
+import { useTranslation } from 'contexts/Localization'
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledRibbon = styled.div<{ color: string }>`
+  display: flex;
+  justify-content: center;
+  background-color: ${({ theme, color }) => theme.colors[color]};
+  color: white;
+  position: absolute;
+  right: 0;
+  top: 0;
+  text-align: center;
+  width: 94px;
+  height: 94px;
+  clip-path: polygon(0 0, 40% 0, 100% 60%, 100% 100%);
+
+  & > div {
+    padding-top: 23%;
+    overflow: hidden;
+    transform: rotate(45deg);
+  }
+`
+
+const AuctionRibbon: React.FC<{ auction: Auction; noAuctionHistory: boolean }> = ({ auction, noAuctionHistory }) => {
```

Comment:
Had to make special ribbon for it instead of using one in UIKit cause the one it UIKit couldn't be positioned properly in this case (since AuctionLeaderboard is not normal card)